### PR TITLE
No more posts just about us page

### PR DESCRIPTION
**Pull request by geshan**

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -1,8 +1,24 @@
 ---
 layout: page
-title: About
+title: About PHP Developers Community, Nepal
```
Comment (hardfire) [inline about.md:3]: can we make this shorter, because it shows up in the top bar as well ?

Comment (geshan) [inline about.md:3]: what do you suggest?

Comment (geshan) [inline about.md:3]: I think its fine @hardfire

Comment (hardfire) [inline about.md:3]: ok

**Pull request merged**